week-1/mandatory/2-classes-db/task.md

```diff
@@ -7,8 +7,28 @@ Below you will find a set of tasks for you to complete to consolidate and extend
 To submit this homework write the correct commands for each question here:
 
 ```sql
-
-
+--1
+select room_no from rooms where rate>100;
+--2
+select * from reservations where  (checkin_date  between '2020-09-01' and '2020-09-30')and ( checkout_date- checkin_date >3);
+--3
+select * from customers where city like 'M%';
+--4
+INSERT INTO room_types VALUES ('PENTHOUSE',185.00);
+--5
+INSERT INTO rooms   select 501 as room_no, def_rate,'PENTHOUSE' as room_type,null as no_guest from room_types where room_type='PENTHOUSE';
+
+INSERT INTO rooms   select 502 as room_no, def_rate,'PENTHOUSE' as room_type,null as no_guest from room_types where room_type='PENTHOUSE';
+--6
+INSERT INTO rooms VALUES (503,143.00,'PREMIER PLUS',null);
+--7
+select count(room_no) from reservations where  (checkin_date  between '2020-08-01' and '2020-08-31');
```

Comment:
Good - you could have used `count(*)` but there's no practical difference with these.  To catch the cases where the room is occupied across the ends of the month you might try:
`... WHERE checkin_date <= '2020-08-31' AND checkout_date > '2020-08-01'`